task/06-conditions-n-loops-tasks.js

```diff
@@ -81,8 +94,8 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,1,1   =>  false
  *   10,10,10 =>  true
  */
-function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+function isTriangle(a, b, c) {
+    return ((a + b > c) && (a + c > b) && (b + c > a)) ? true : false;
```

Comment:
We can just return ((a + b > c) && (a + c > b) && (b + c > a)) with the same result
Please, see https://javascript.info/logical-operators